inst/examples/installGithub.r

```diff
@@ -39,4 +40,6 @@ if (opt$deps == "TRUE" || opt$deps == "FALSE") {
     opt$deps <- NA
 }
 
-invisible(sapply(opt$REPOS, function(r) install_github(r, dependencies = opt$deps)))
+opt$update <- as.logical(opt$update)
```

Comment:
That looks to be key.  

I presume you tested again default, set, not set, crazy text input, ... and it reduces correctly to just `TRUE` or `FALSE`?